src/components/TextInput.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import TextField from 'material-ui/TextField';
+import PropTypes from 'prop-types';
+
+function RenderTextField({ ...props }) {
+  const textProp = Object.assign({}, props);
+  delete textProp.meta;
+  delete textProp.input;
+
+  return (
+    <TextField
+      {...textProp}
+    />
+  );
+}
+const TextInput = ({ name, ...props }) => (
+  <Field
+    {...props}
+    name={name}
+    id={name}
```

Comment:
`name` is, afaik, only unique within the form. An html `id` attribute needs to be globally unique. If I'm not mistaken, setting `id` like this will cause problems if you have two forms active that both have a field named `address` (for example).

Do we need to set an `id` attribute?